test/simplifyExpression/simplify.test.js

```diff
@@ -91,6 +91,18 @@ describe('distribution', function () {
     ['(x-2)(x-4)', 'x^2 - 6x + 8'],
     ['- x*y^4 (6x * y^2 + 5x*y - 3x)',
       '-6x^2 * y^6 - 5x^2 * y^5 + 3x^2 * y^4'],
+    ['(nthRoot(x, 2))^2', 'x'],
+    // TODO: fix nthRoot to evaluate nthRoot(x^3, 2)
+    // ['(nthRoot(x, 2))^3', 'nthRoot(x ^ 3, 2)'],
+    // ['3 * nthRoot(x, 2) * (nthRoot(x, 2))^2', '3 * nthRoot(x ^ 3, 2)'],
+    // TODO: fix when base has multiplication
+    // ['(nthRoot(x, 2) * nthRoot(x, 2))^2', 'x^2'],
```

Comment:
Not exactly. This is because distribution code runs before function (nthRoot) search. So we would do `(nthRoot(x^2 , 2)^2` -> `nthRoot(x^2, 2)` * `nthRoot(x^2, 2)`.